ideas/167-extended-automated-testing-for-continuous-delivery/README.md

```diff
@@ -0,0 +1,58 @@
+## Preamble
+
+    Idea: 167
+    Title: Functional automated testing as a part of Continuous Delivery process
+    Status: Draft
+    Created: 2018-04-12
+
+## Summary
+For automated testing to be a part of CD process we need to:
+- increase coverage
+- increase reliability
+- provide visible results on daily basis
+
+## Swarm Participants
+- Lead Contributor: @antdanchenko
+- Testing & Evaluation: @antdanchenko
+- Contributor: @yevh-berdnyk
+- Contributor: @lukaszfryc
+- PM:
+
+## Product Overview
+Extended set of automated tests which will run against each nightly build in order to provide results via TestRun in TestRail + short summary via Slack notification
+As result during nightly build testing a tester should execute only those tests which are not covered by automation in the same TestRun
+In case of failure in automated test a tester should add an appropriate status to the test (see `An option to review results should be added for each test via adding 4 statuses to TestRail` in `Product Description` section)
```

Comment:
Is the review going to happen before the results are shared with others using Slack notification?

Reply:
review is going to happen after, Slack notification is just a summary with number of failed/passed tests